cmd/mesh/manifest-diff.go

```diff
@@ -33,11 +33,23 @@ const YAMLSuffix = ".yaml"
 type manifestDiffArgs struct {
 	// compareDir indicates comparison between directory.
 	compareDir bool
+	// selection specifies the resources to compare, it's comma-separated of resource indicators
+	// in the form of "<kind>:<namespace>:<name>,<kind>:<namespace>:<name>".
+	selection string
+	// ignore specifies the resources to ignore, it's comma-separated of resource indicators
+	// in the form of "<kind>:<namespace>:<name>,<kind>:<namespace>:<name>".
+	ignore string
 }
 
 func addManifestDiffFlags(cmd *cobra.Command, diffArgs *manifestDiffArgs) {
 	cmd.PersistentFlags().BoolVarP(&diffArgs.compareDir, "directory", "r",
 		false, "compare directory")
+	cmd.PersistentFlags().StringVar(&diffArgs.selection, "select", "::",
+		"specifies the resources to compare, it's comma-separated of resource indicators"+
+			"in the form of \"<kind>:<namespace>:<name>,<kind>:<namespace>:<name>\".")
```

Comment:
Use the text above, if it looks good to you.

Reply:
done.